account_companyweb/model/res_partner.py

```diff
@@ -37,6 +37,8 @@ class res_partner(orm.Model):
     _inherit = 'res.partner'
 
     def companyweb_information(self, cr, uid, ids, vat_number, context=None):
+        if not context:
+            context = {}
```

Comment:
context is being returned at the end of the method. When calling it over xmlrpc, it can't be None as None can't be marshalled.